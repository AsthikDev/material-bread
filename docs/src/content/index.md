---
path: /
layout: index
title: Material Bread
description: Highly Customizable Material Design Components
---
