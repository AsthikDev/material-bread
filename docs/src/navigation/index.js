import componentPages from './componentPages';
import contributingPages from './componentPages';
import gettingStartedPages from './gettingStartedPages';
import stylePages from './stylePages';
import utilsPages from './stylePages';

export {
  componentPages,
  contributingPages,
  gettingStartedPages,
  stylePages,
  utilsPages,
};
