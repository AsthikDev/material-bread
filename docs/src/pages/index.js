import React from 'react';

import Layout from '../components/Layout/Layout';
import { Button } from '@material-ui/core';

const IndexPage = ({ data }) => {
  return (
    <Layout>
      <h1 className="Home__Title">Material Bread</h1>
      <h4 className="Home__subtitle">
        React Native Material Design Components
      </h4>

      <div className="row center-xs" style={{ marginTop: 40 }}>
        <Button variant="outlined" color="primary">
          Get Started
        </Button>

        <Button
          variant="outlined"
          color="secondary"
          style={{ marginLeft: 16, marginRight: 16 }}>
          Playground
        </Button>

        <Button variant="outlined" color="default">
          Github
        </Button>
      </div>
    </Layout>
  );
};

export default IndexPage;
