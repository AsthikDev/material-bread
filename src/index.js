// ==== Abstract ====
export { default as Ripple } from './Abstract/Ripple';
export { default as shadow } from './Abstract/shadow';

// ==== Components ====
export { default as Appbar } from './Components/Appbar';
export { default as Avatar } from './Components/Avatar';
export { default as Badge } from './Components/Badge';
export { default as Banner } from './Components/Banner';
export { default as Button } from './Components/Button';
export { default as Card } from './Components/Card/Card';
export { default as CardActions } from './Components/Card/CardActions';
export { default as CardContent } from './Components/Card/CardContent';
export { default as CardHeader } from './Components/Card/CardHeader';
export { default as CardMedia } from './Components/Card/CardMedia';
export { default as Checkbox } from './Components/Checkbox';
export { default as Chip } from './Components/Chip';
export { default as Dialog } from './Components/Dialog/Dialog';
export { default as DialogActions } from './Components/Dialog/DialogActions';
export {
  default as DialogContentText,
} from './Components/Dialog/DialogContentText';
export { default as DialogTitle } from './Components/Dialog/DialogTitle';
export { default as Dropdown } from './Components/Dropdown';
export { default as Divider } from './Components/Divider';
export { default as Icon } from './Components/Icon';
export { default as Menu } from './Components/Menu';
export { default as MenuItem } from './Components/MenuItem';
export { default as Paper } from './Components/Paper';
export {
  Heading,
  Subtitle,
  BodyText,
  Caption,
  Overline,
} from './Components/Typography';

// ==== Theme ====
export { default as ThemeContext } from './Theme/ThemeContext';
export { default as withTheme } from './Theme/withTheme';
