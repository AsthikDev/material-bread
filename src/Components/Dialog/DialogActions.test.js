import React from 'react';
import { DialogActions } from '../../../src';

import renderer from 'react-test-renderer';

test('DialogActions Renders', () => {
  const tree = renderer.create(<DialogActions />).toJSON();
  expect(tree).toMatchSnapshot();
});
