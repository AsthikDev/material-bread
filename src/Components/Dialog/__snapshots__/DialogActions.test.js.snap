// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogActions Renders 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "flex-end",
      "marginBottom": 8,
      "paddingHorizontal": 8,
    }
  }
/>
`;
